common/src/KokkosFFT_utils.hpp

```diff
@@ -8,19 +8,60 @@
 #include <Kokkos_Core.hpp>
 #include <vector>
 #include <set>
+#include <string>
 #include <algorithm>
 #include <numeric>
 #include "KokkosFFT_traits.hpp"
 
+#ifndef KOKKOS_ENABLE_CXX17
```

Comment:
```suggestion
#if defined(KOKKOS_ENABLE_CXX17)
```
Will be easier to extend to other conditions.